### PR TITLE
Issue 1733 / 1735 - Eliminate special casing for PMs to minor accounts

### PR DESCRIPTION
**Pull request by hotlevel4**
Changed minor accounts to receive PMs from all registered users.
Fixed call when checking whether a PM can be sent.

Fixes issue #1733. Fixes issue #1735.

### CODE REVIEW

File: cgi-bin/LJ/Setting/UserMessaging.pm [hunk 1]
```diff
@@ -32,7 +32,7 @@ sub option {
     my ($class, $u, $errs, $args) = @_;
     my $key = $class->pkgkey;
 
-    my $usermsg = $class->get_arg($args, "usermsg") || $u->prop("opt_usermsg");
+    my $usermsg = $class->get_arg($args, "usermsg") || $u->opt_usermsg);
```
Comment (kareila) [inline cgi-bin/LJ/Setting/UserMessaging.pm]: Oops! It looks like there is a stray close paren here that also needs to be removed.

**Comment by kareila** [comment]
One trivial syntax fix before this can go in.  Thank you!

**Comment by hotlevel4** [comment]
Argh, sorry, fixed.

**Comment by kareila** [comment]
No worries :)

**Pull request merged**